python/pyproject.toml

```diff
@@ -64,6 +64,7 @@ tests = [
 dev = ["ruff==0.4.1"]
 benchmarks = ["pytest-benchmark"]
 torch = ["torch"]
+cuvs = ["cuvs-cu12", "pylibraft-cu12"]
```

Comment:
do we need to limit the platform to linux only? Will it fail on mac / windows?

Reply:
I don't have a windows/mac machine to test on, but this does require a cuda device. It may be worth testing on windows in the future.